### ISSUE TITLE
Repeating Error Bar

### ISSUE THREAD

**Issue by temuller765** [ringcentral/hubspot-embeddable-ringcentral-phone#277]
![Screen Shot 2021-11-10 at 10 58 17 AM](https://user-images.githubusercontent.com/62224843/141168412-e06d0d6b-19e9-422e-91a8-9f351fd7cbbe.png)

I keep reconnecting to the Hubspot account, but it does not fix the issue

**Comment by zxdong262** [comment]
@temuller765 Have deployed a fix, should not show duplicate error notification now

**Comment by Ruba85** [comment]
Yes we have the same issues. And now we don't see  notification.
But we don't see the call button too and there is some errors in console.
![no call 2](https://user-images.githubusercontent.com/58745486/141297771-223b48e7-b25e-46fa-900d-8fe318e47343.png)
![no call](https://user-images.githubusercontent.com/58745486/141297777-c3a27556-2a4e-4148-972a-d10bd9e77789.png)

**Comment by zxdong262** [comment]
> Yes we have the same issues. And now we don't see notification. But we don't see the call button too and there is some errors in console. ![no call 2](https://user-images.githubusercontent.com/58745486/141297771-223b48e7-b25e-46fa-900d-8fe318e47343.png) ![no call](https://user-images.githubusercontent.com/58745486/141297777-c3a27556-2a4e-4148-972a-d10bd9e77789.png)

That would be hitting hubspot api rate limit, currently we are working on a solution for this.

**Comment by Ruba85** [comment]
Hello. Is there any update about this issues?

**Comment by temuller765** [comment]
Hi ,

This is no longer an issue. Thank you for your help!

**Comment by Ruba85** [comment]
Unfortunately we still have this problem
![image](https://user-images.githubusercontent.com/58745486/141798661-f5812e5f-4459-43a0-825a-dceb8f3e8bf0.png)

**Comment by zxdong262** [comment]
@Ruba85 Turns out HubSpot make the token expire time shorter, now is only 30 minutes, I have deployed a update to address this change, please try again to see if this fixed it. This is where I found it: https://community.hubspot.com/t5/Developer-Announcements/Upcoming-Expiration-of-OAuth-access-tokens-is-changing/td-p/497731

**Comment by Ruba85** [comment]
Thank you. For now it seems that everything work fine.
